insalan/tournament/models.py

```diff
@@ -857,6 +861,77 @@ class Caster(models.Model):
     )
 
 
+class TournamentMailer(models.Model):
```

Comment:
This could have a doc string. What is the purpose of having a model related to the mailer exactly? What does each entry represent?

Reply:
This model is only here to create an admin panel view. I tried using a model with "managed=False" so that no table was created by django but this was causing troubles because of the foreign key (even if we say "on_delete=models.SET_NULL") on tournament deletion.
I'll add a doc string on this tho.